packages/teleport/src/config.ts

```diff
@@ -108,7 +111,8 @@ const cfg = {
     nodeTokenPath: '/v1/enterprise/nodes/token',
     nodeScriptPath: '/scripts/:token/install-node.sh',
     appNodeScriptPath: '/scripts/:token/install-app.sh?name=:name&uri=:uri',
-    requestAccessPath: '/v1/enterprise/accessrequest/:requestId?',
+    accessRequestPath: '/v1/enterprise/accessrequest/:requestId?',
```

Comment:
```
accessRequestPath: '/v1/enterprise/accessrequest/:requestId?',
accessRequestsPath: '/v1/enterprise/accessrequest?id=:requestId?',
```

we are not going to open source these, so lets move it to E config. The reason why some of these endpoints reside in OSS is becauase roles and users are scheduled to be open sourced soon. But not the workflows.

Reply:
```
accessRequestPath: '/v1/enterprise/accessrequest/:requestId?',
accessRequestsPath: '/v1/enterprise/accessrequest?id=:requestId?',
```

Why are there 2 ways to pass a request ID?

Reply:
moved to enterprise and deleted additional path